civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -5486,6 +5488,50 @@ function civihr_employee_portal_form_views_exposed_form_alter(&$form, &$form_sta
             }
         }
     }
+
+    if (substr($form['#id'], 0, 33) === 'views-exposed-form-civihr-report-') {
+        foreach ($form as $key => $value) {
+          if (isset($form['#info']["filter-$key"])) {
+            $form[$key]['value'] = generateFilterInputStructure($form['#info']["filter-$key"]['label']);
```

Comment:
I understand that you specifically point the `min` and `max` fields with the add-datepicker-button function, but what about `$form['#info']["filter-$key"]`? Is it safe to assume that any `filter-` field needs to have a datepicker?